content/docs/configuration/basics/configuration-mapping.md

```diff
@@ -177,17 +177,17 @@ The concept also extends to 1D-2D, 2D-3D, and further setups, which are not curr
 Potential configurations for the axial and radial geometric multiscale mapping look as follows:
 
 ```xml
-<mapping:axial-geometric-multiscale direction="read" multiscale-type="spread" multiscale-radius="1.0" multiscale-axis="X" from="MyMesh2" to="MyMesh1" constraint="consistent" />
+<mapping:axial-geometric-multiscale direction="read" multiscale-type="spread" multiscale-radius="1.0" multiscale-axis="X" spread-profile="parabolic" from="MyMesh2" to="MyMesh1" constraint="consistent" />
 ```
 
 ```xml
 <mapping:radial-geometric-multiscale direction="read" multiscale-type="collect" multiscale-axis="X" from="MyMesh1" to="MyMesh2" constraint="consistent" />
 ```
 
 The `multiscale-type` which can be either `"spread"` or `"collect"` refers to whether the participant spreads data from one mesh node to multiple nodes or collects data from multiple mesh nodes into one node. The `multiscale-axis` is the main axis, along which the coupling takes place, i.e. the principal axis of the 1D and 3D participants. The `multiscale-radius` refers to the radius of the circular interface boundary surface.
+When using a `"spread"` mapping, the 1D quantity is distributed over the 3D interface using the `spread profile`, which defines how the value varies radially across the circular interface. It can either be "uniform", where the same value is applied to all interface vertices; or "parabolic", in which a parabolic profile is applied (maximum at the centerline, decreasing to zero at the wall).
```

Comment:
I would highlight the options in code, as they are clearer to understand that they are verbatim, and easier to spot:


```suggestion
When using a `"spread"` mapping, the 1D quantity is distributed over the 3D interface using the `spread-profile`, which defines how the value varies radially across the circular interface. It can either be `"uniform"`, where the same value is applied to all interface vertices; or `"parabolic"`, in which a parabolic profile is applied (maximum at the centerline, decreasing to zero at the wall).
```

The quotes are optional, and I would tend to remove them, but since the rest of the section has them already, let's stay consistent in this PR.